app/controllers/api/v1/projects_controller.rb

```diff
@@ -62,6 +62,12 @@ def update
       # transaction.
       content_attributes = primary_content_attributes(update_params)
       unless content_attributes.blank?
+        resource.display_name = content_attributes[:title] if content_attributes.key?(:title)
```

Comment:
I'm not super happy with how this is set up. But couldn't come up with something nicer that worked. Let me hear suggestions if you have m